### DISCUSSION TITLE
PRO-HD Component 1920x544 streaming output resolution question

### DISCUSSION THREAD

**Discussion by Chaskel** [GerryDazoo/Slinger#260] [Q&A] [answered]
Hello,

My understanding from the old https://www.slingbox.com/help/KB/KB-2000464 page is that a PRO-HD supports 1920x544 output. In the table they had it listed as 1920x544** with the ** indicating "30 fps and 1/2 1080i resolution (doubled to a 1080i resolution)".

When I set resolution to 15 (SBVS_1920x544) in the config.ini file, VLC playback codec information shows the resolution as 1920x544 and the image is squished with a black bar across the top and bottom. My incoming source is 1920x1080i/60 (using one of these https://www.amazon.com/dp/B09L8FZHJW), and I was wondering if the ** note of that KB article is being taken into account by slinger?

In case the following is helpful, the following link is where I came across that 1920x540 relates to field size as opposed to 1080i frame size: https://www.avsforum.com/threads/1920-x-540-resolution.778139/ (also referenced in post 3118 here: https://www.monitortests.com/forum/Thread-Custom-Resolution-Utility-CRU?pid=7225#pid7225).

Thank you

**Comment by GerryDazoo** [comment, marked as answer]
I have no control over what the slingbox sends. All I do is forward the video packets from the slingbox to your media player. Use the aspect ratio controls on your media player to set it to 16:9. 
Why don't you set your resolution to 16 (1920x1080) then you should receive a 1920x1080 stream instead of 1920x544.

**Reply by Chaskel** [reply]
Thank you for the quick response. I had originally tried with 16, but then went to 15 so that I would match what the official spec showed. Per your suggestion I have just tried again using both 15 and 16 with the aspect ratio controls set to 16:9, and it appears to address the problem. For some reason there is a small yellow line running across the top of the video stream, but other than that it seems to do the trick.

**Reply by sp222222** [reply]
i put mine on 16 with  my PRO HD and all I get is  x544 but it's not 1920 using 16 that I'm for sure of. it looks fantastic regardless.
